jenkins-integration/config/nodes/example-node1.json

```diff
@@ -11,14 +11,14 @@
             "path": "/root/test-catalogs/catalog_zero/modules/catalog_zero1"
```

Comment:
These modules are a little easier to work with during development as they have default values for class parameters.
Nothing is using these still, just easier for development.